code/modules/modular_computers/computers/item/pda.dm

```diff
@@ -140,6 +140,12 @@
 
 	return . || NONE
 
+/obj/item/modular_computer/pda/interact_with_atom(atom/interacting_with, mob/living/user, list/modifiers)
+	if(iscash(interacting_with))
+		money_act(user, interacting_with)
+		return ITEM_INTERACT_SUCCESS
```

Comment:
`money_act(...)` already returns interaction bitflags, better to use those than to assume it's a success
```suggestion
		return money_act(user, interacting_with)
```